Source/src/WixSharp/UninstallFullUI.cs

```diff
@@ -26,22 +26,36 @@ public static void EnableUninstallFullUI(this Project project)
             EnableUninstallFullUI(project, null);
         }
 
+        /// <summary>
+        /// Enables the full UI.
+        /// </summary>
+        /// <param name="project">The project.</param>
+        /// <param name="logFilePath">Path for MSI logs file</param>
+        /// <param name="logSwitches">MSI logging switches</param>
+        public static void EnableUninstallFullUI(this Project project, string logFilePath, MsiexecLogSwitches logSwitches)
```

Comment:
I think the solution is more complicated that it should be. 
I would propose a solution when we have simple "string extraParameters" instead of logFilePath and logSwitches.  
This is going to be call
```
project.EnableUninstallFullUI("[#myapp_exe],0", @"/L*V [TempFolder]CustomMsiLog.log PROP1=VAL1 PROP2=VAL2");
```
It will make the implementation less dependent on MSI version (in case they add more control switches) and will allow to pass arbitrary properties (like PROP1, PROP2) in case it's needed.

Reply:
Okay, will change this according to your suggestion